src/main/java/edu/wisc/my/keyvalue/controller/KeyValueStoreController.java

```diff
@@ -26,6 +25,18 @@ public void setKeyValueService(IKeyValueService keyValueService){
         this.keyValueService = keyValueService;
     }
     
+    @RequestMapping("/")
+    public @ResponseBody void index(HttpServletRequest request, HttpServletResponse response){
+        try {
+            response.getWriter().write("Hello World");
+            response.setContentType("application/json");
+            response.setStatus(HttpServletResponse.SC_OK);
+        } catch (IOException e) {
+            // TODO Auto-generated catch block
+            e.printStackTrace();
+        }
+    }
+    
     
```

Comment:
Thought. Change this to a status response. We could then use this to verify the service is alive.